src/transactions/feed/queryHelper.ts

```diff
@@ -187,8 +188,7 @@ export function useFetchTransactions(): QueryHookResult {
   // Query for more transactions if requested
   useAsync(
     async () => {
-      if (!fetchingMoreTransactions || !anyNetworkHasMorePages(fetchedResult.pageInfo)) {
```

Comment:
Removing `!fetchingMoreTransactions` allows tests to pass, but can result in multiple fetch requests firing. Should likely be included and tests updated.